examples/polkadot/analyze-jam.py

```
@@ -0,0 +1,171 @@
+        data = f.read()
+
+    off = 0
+    # Metadata prefix: varint length + metadata bytes
+    meta_len, n = read_var_u32(data, off); off += n
+    off += meta_len
+    ro_len = int.from_bytes(data[off:off+3], "little"); off += 3
+    rw_len = int.from_bytes(data[off:off+3], "little"); off += 3
+    heap_pages = int.from_bytes(data[off:off+2], "little"); off += 2
+    stack_size = int.from_bytes(data[off:off+3], "little"); off += 3
+    off += ro_len + rw_len  # skip ro/rw data
+    code_total_len = int.from_bytes(data[off:off+4], "little"); off += 4
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)